src/locales/en_US.js

```diff
@@ -720,7 +721,7 @@ const strings = {
   fio_request_by_fio_address_error_invalid_amount: 'Please enter an amount before requesting with FIO',
   fio_request_error_header: 'Request Failed',
   fio_request_ok_body: 'Your request has been successfully sent',
-  fio_request_add_bundles_ok_text: 'Bundles has been successfully purchased',
+  fio_request_add_bundles_ok_text: 'Bundled transactions have been successfully purchased',
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name